imports/ui/explorer-components/sort-items.js

```diff
@@ -7,6 +7,13 @@ export const SORT_BY = {
   OLDEST_UPDATE: 5
 }
 
+export const labels = [
+  [SORT_BY.DATE_DESCENDING, {category: 'Newest (Created)', selected: 'Created ↓'}],
+  [SORT_BY.DATE_ASCENDING, {category: 'Oldest (Created)', selected: ' Created ↑'}],
```

Comment:
can we replace
category: 'Oldest (Created)'
with
category: 'Created - Oldest'